packs/data/feats.db/cooperative-nature.json

```diff
@@ -19,7 +19,17 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "all",
```

Comment:
This is going to appear in damage rolls as well

Reply:
We have a selector for everything but damage rolls?

Reply:
Updated